ci-operator/config/openshift/hive/openshift-hive-mce-2.3.yaml

```diff
@@ -0,0 +1,212 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  openshift_release_rhel-8-release-golang-1.19-openshift-4.14:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.20-openshift-4.14
```

Comment:
ntr: these match the dockerfile currently at the tip of the mce-2.3 branch. OCP version bumped everywhere _except_...